src/pages/Home.jsx

```diff
@@ -1,6 +1,92 @@
-import { PageContainer } from '../components';
+import styled from 'styled-components';
+import { PageContainer, ButtonCta, NotesModal } from '../components';
+import FilterAltOutlinedIcon from '@mui/icons-material/FilterAltOutlined';
+import { useEffect, useState } from 'react';
+import { NotesdModal } from '../components/NotesModal';
+import AddIcon from '@mui/icons-material/Add';
+import NoteAddOutlinedIcon from '@mui/icons-material/NoteAddOutlined';
+import { useModal } from '../contexts/ModalContext';
+import { useData } from '../contexts/DataContext';
+import { NotesContainer, NoteCard } from './NoteCard';
 export const Home = () => {
-  return (<PageContainer >
-    Home Page work in Progress
+  const { isNotesModalOpen, setIsNotesModalOpen } = useModal();
+  const { notes, currentNote, setCurrentNote } = useData();
+  const pinnedNotes = notes.filter((note) => note.isPinned === true);
+  const unPinnedNotes = notes.filter((note) => note.isPinned === false);
+  useEffect(() => {
+    window.scrollTo(0, 0);
+
+  })
+  return (<PageContainer isNotesModalOpen={isNotesModalOpen}>
+    <NotesHeader >
+      <HeadingLarge>Notes</HeadingLarge>
+      <PageActions>
+        <FilterButton><FilterAltOutlinedIcon /><span>Filter</span></FilterButton>
+        <ButtonCta onClick={() => { setIsNotesModalOpen(true) }}><AddIcon /> <span>Add Note</span></ButtonCta>
+      </PageActions>
+    </NotesHeader>
+    {/* <NotesdModal isModalOpen={isNotesModalOpen} setIsModalOpen={setIsNotesModalOpen} note={currentNote} setNote={setCurrentNote} /> */}
```

Comment:
You can remove these comments if no longer needed

Reply:
removed it , thanks